test/surface_test.py

```diff
@@ -1198,6 +1198,21 @@ def test_convert_alpha_SRCALPHA(self):
         finally:
             pygame.display.quit()
 
+    def test_convert_get_alpha(self):
+        """ converted()d surfaces loaded from png transparent style surfaces should return 255.
+        """
+        pygame.display.init()
+        try:
+            pygame.display.set_mode((640, 480))
+
+            s1 = pygame.Surface((100, 100), pygame.SRCALPHA, 32)
+            s1_converted = s1.convert()
+            self.assertEqual(s1_converted.get_flags() & SRCALPHA, SRCALPHA)
```

Comment:
```suggestion
            self.assertEqual(s1_converted.get_flags() & SRCALPHA, 0)
```

The converted surface does not use SRCALPHA blends anymore or does it?
According to docs:
`SRCALPHA       0x00010000    # Blit uses source alpha blending`

Reply:
I this flag managed by SDL?